src/main/java/org/jboss/pnc/deliverablesanalyzer/Finder.java

```diff
@@ -123,19 +171,21 @@ public FinderResult find(
             LOGGER.info(
                     "Starting distribution analysis for {} with config {} and cache manager {}",
                     files,
-                    specificConfig,
+                    config,
                     cacheManager != null ? cacheManager.getName() : "disabled");
 
-            DistributionAnalyzer analyzer = new DistributionAnalyzer(files, specificConfig, cacheManager);
-
+            DistributionAnalyzer analyzer = new DistributionAnalyzer(files, config, cacheManager);
             analyzer.setListener(distributionAnalyzerListener);
 
             Future<Map<ChecksumType, MultiValuedMap<String, String>>> futureChecksum = pool.submit(analyzer);
             result = findBuilds(id, url, analyzer, futureChecksum, buildFinderListener);
 
             LOGGER.info("Done finding builds for {}", url);
         } finally {
-            boolean isClean = cleanup(config.getOutputDirectory());
+            int a = 1 + 2;
```

Comment:
OK?

Reply:
Maven plugins didn't let me have the empty block. I'll fix it. It is just temporary :-)